executor/runtime/docker/capabilities.go

```diff
@@ -30,36 +30,49 @@ func addAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.Hos
 }
 
 func setupAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.HostConfig) error {
-	addedCapabilities := addAdditionalCapabilities(c, hostCfg)
+	if c.TitusInfo.GetAllowNestedContainers() {
+		return errors.New("nested containers no longer supported")
+	}
 
-	// Privileged containers automaticaly deactivate seccomp and friends, no need to do this
 	fuseEnabled, err := c.GetFuseEnabled()
+	if err != nil {
+		return err
+	}
 
+	kvmEnabled, err := c.GetKvmEnabled()
```

Comment:
Do KVM containers need CAP_SYS_ADMIN?

Reply:
no, not to launch the VM anyway post network configuration

Reply:
I think for the network config, all you need is CAP_NET_ADMIN.

Reply:
Yes